lib/ObjWriter/jitDebugInfo.h

```diff
@@ -0,0 +1,46 @@
+#ifndef JIT_DEBUG_INFO_H
+#define JIT_DEBUG_INFO_H
+
+typedef unsigned int DWORD;
+#define _TARGET_AMD64_ 1
+
```

Comment:
Looks like LLILCPal.h doesn't contain the definition of DWORD. Removing "typedef unsigned int DWORD;" gives :
D:\git\llilc\lib\ObjWriter\objwriter.cpp(277): error C2589: 'constant': illegal token on right side of '::' [d:\build\llilc-build\lib\ObjWriter\objwriter.vcxproj]
D:\git\llilc\lib\ObjWriter\objwriter.cpp(277): error C2059: syntax error: '::' [d:\build\llilc-build\lib\ObjWriter\objwriter.vcxproj]
D:\git\llilc\lib\ObjWriter\objwriter.cpp(280): error C2589: 'constant': illegal token on right side of '::' [d:\build\llilc-build\lib\ObjWriter\objwriter.vcxproj]
D:\git\llilc\lib\ObjWriter\objwriter.cpp(280): error C2059: syntax error: '::' [d:\build\llilc-build\lib\ObjWriter\objwriter.vcxproj]